Qiqqa/Common/BackgroundWorkerDaemonStuff/BackgroundWorkerDaemon.cs

```diff
@@ -58,15 +57,6 @@ private void DoMaintenance_OnceOff(Daemon daemon)
 
                 InitClientUpdater();
 
-                try
-                {
-                    AlternativeToReminderNotification.CheckIfWeWantToNotify();
-                }
-                catch (Exception ex)
-                {
-                    Logging.Error(ex, "Exception during AlternativeToReminderNotification.CheckIfWeWantToNotify");
-                }
-

```

Comment:
I had not removed these as I was wondering how to "market" the new Qiqqa: how to enable people to find it when they don't know the name yet but are looking for something like it. 

Having a ranking at alterative.to and similar sites can help to raise awareness. I'm not a marketeer and lack better ideas than what was already implemented (yup, the twitter nags were indeed nags, but there's a balance to strike: currently Qiqqa is the most *unknown* creature out there when compared to the big boys like Zotero, Mendeley, et al -- while those serve partly different purposes, there's plenty of overlap.

For that reason, I'm inclined to keep this around in Qiqqa, despite its 'obnoxiousness'...

Better ideas welcome (might want to file them in an issue as that's a far better place for this higher-level discussion than code comments. 😄

Reply:
I think perfect place for such information would be qiqqa.org webpage, release pages and "Tutorials & Help" panel on "Home (F1)" view.
Then again, a little nagging once in a while doesn't hurt, especially when it goes away for 12 months if you click on the link, regardless if you vote or not.
Reverted.